src/Service/TwitterService.php

```diff
@@ -34,20 +36,23 @@ private function get($request, $params = []) {
         return $result;
     }
 
-    public function getTweetsSince($last_id) {
+    public function getTweetsSince($last_id)
+    {
         return $this->get('https://api.twitter.com/1.1/statuses/user_timeline.json', [
```

Comment:
Wouldn't be a bad idea have the base url in a single spot, makes it a bit easier to maintain.